src/main/kotlin/com/exactpro/th2/common/schema/grpc/configuration/GrpcConfiguration.kt

```diff
@@ -46,7 +47,8 @@ data class GrpcClientConfiguration(
 data class GrpcEndpointConfiguration(
     @JsonProperty(required = true) var host: String,
     @JsonProperty(required = true) var port: Int = 8080,
-    var attributes: List<String?> = emptyList()
+    @JsonProperty var maxMessageSize: Int = GrpcUtil.DEFAULT_MAX_MESSAGE_SIZE,
```

Comment:
maxMessageSize should be use for client and server, also, users use only default value for this option I think.
Let's move this option from EndpointConfiguration to the GrpcConfiguration class